pallet-logion-loc/src/fees.rs

```diff
@@ -94,11 +94,18 @@ pub struct Fees {
     pub legal_fee_beneficiary: Option<Beneficiary<AccountId>>,
     pub certificate_fees: Balance,
     pub value_fee: Balance,
+    pub collection_item_fee: Balance,
+    pub tokens_record_fee: Balance,
```

Comment:
Note that the business rather talks about "recurrent fees" so using `item_recurrent_fee` and `record_recurrent_fee` would be closer to what they expect.